src/components/Header/SearchBar/Search.module.scss

```diff
@@ -0,0 +1,31 @@
+.searchBarContainer {
+  display: flex;
+  align-items: center;
+  font-size: 1.5rem;
+  line-height: 1.2rem;
+}
+  
+.searchIcon {
+  color: var(--text-color);
+}
+
+.searchIcon:hover {
+  cursor: pointer;
+  color: var(--text-hover-color);
+}
+
+.expandedSearchBar {
```

Comment:
The search bar has the same color on dark and light theme:
![image](https://github.com/cthit/chalmers.it/assets/7264745/99b0f3f7-c581-42ea-8743-6556e4c1ef07)
![image](https://github.com/cthit/chalmers.it/assets/7264745/12c7cce5-2f8c-4923-82c2-d2ae00e54326)
Disregard this if it's intended, but I would consider using a less contrasting color on light theme!

Reply:
They were both white for me so it probably followed the light/dark mode of the system, the search bar and text follow the theme of the site now, I added a border too.